auto-generated/cluster/stone-prd-rh01/tenants/multiarch-tuning-ope-tenant/projctl.konflux.dev_v1beta1_projectdevelopmentstreamtemplate_file-based-catalog.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: projctl.konflux.dev/v1beta1
+kind: ProjectDevelopmentStreamTemplate
+metadata:
+  name: file-based-catalog
+  namespace: multiarch-tuning-ope-tenant
+spec:
+  project: file-based-catalog
+  resources:
+  - apiVersion: appstudio.redhat.com/v1alpha1
+    kind: Application
+    metadata:
+      annotations:
+        application.thumbnail: "6"
+        finalizeCount: "0"
+      finalizers:
+      - spi.appstudio.redhat.com/remote-secrets
+      - application.appstudio.redhat.com/finalizer
+      name: fbc-{{.versionName}}
+    spec:
+      appModelRepository:
+        url: ""
+      displayName: fbc-{{.versionName}}
+      gitOpsRepository:
+        url: ""
+  - apiVersion: appstudio.redhat.com/v1alpha1
+    kind: Component
+    metadata:
+      annotations:
+        applicationFailCounter: "0"
+        build.appstudio.openshift.io/pipeline: '{"name":"fbc-builder","bundle":"latest"}'
+      finalizers:
+      - component.appstudio.redhat.com/finalizer
+      - test.appstudio.openshift.io/component
+      - image-controller.appstudio.openshift.io/image-repository
+      - image-registry-secret-sa-link.component.appstudio.openshift.io/finalizer
+      - pac.component.appstudio.openshift.io/finalizer
+      name: fbc-v4-16
+      namespace: multiarch-tuning-ope-tenant
+    spec:
+      application: fbc-{{.versionName}}
+      componentName: fbc-{{.versionName}}
+      replicas: 0
+      resources:
+        requests:
+          cpu: 100m
+          memory: 512Mi
+      source:
+        git:
+          context: ./
+          devfileUrl: https://raw.githubusercontent.com/openshift/multiarch-tuning-operator/fbc-4.16/devfile.yaml
+          revision: fbc-{{.version}}
+          url: https://github.com/openshift/multiarch-tuning-operator
+      targetPort: 50051
+  variables:
+  - description: A version number for a new OCP stream
+    name: version
+  - description: |
+      The name of some objects cannot include dots. It's recommended to set this versionName replacing any `.` with hyphen (`-`) in the value of the parameter `version`.
+    name: versionName
```

Comment:
You might need an extra newline in the string above this to ensure this is not included in it

Reply:
This should be a rendering issue in the GH UI.

What you read here is generated correctly from https://github.com/redhat-appstudio/tenants-config/pull/475/files#diff-03b792a4bde4e83776f271e135ded766199eac60d2001e0fb41cc1fe29213a83R11